tools/eslint-config-web3-base/ts.js

```diff
@@ -0,0 +1,86 @@
+module.exports = {
+	parser: '@typescript-eslint/parser',
+	plugins: ['@typescript-eslint'],
+	extends: [
+		'airbnb-base',
+		'eslint:recommended',
+		'plugin:@typescript-eslint/recommended',
+		'plugin:@typescript-eslint/recommended-requiring-type-checking',
+		'prettier',
+		'plugin:import/errors',
+		'plugin:import/warnings',
+		'plugin:import/typescript',
+	],
+	rules: {
```

Comment:
Same with these rules

Reply:
I made the rules for the src more strict.